app/assets/javascripts/standalone/progress_tab.js

```diff
@@ -187,7 +187,7 @@ function removeCSSClasses(element, cssClasses) {
 // Daily report
 //
 function updateDailyReport() {
-    const dropdown = document.getElementById("period-dropdown");
+    const dropdown = document.getElementById("daily-period-dropdown");
```

Comment:
This ID also needs to change in the HTML. The dropdowns are still referenced as `period-dropdown` so the daily dropdown is not updating numbers in the report.

Reply:
good catch.